firestore/firestore_quickstart/main.go

```diff
@@ -25,25 +26,28 @@ import (
 	"cloud.google.com/go/firestore"
 )
 
-func main() {
-
-	// [START fs_initialize]
+func createClient(ctx context.Context) *firestore.Client {
 	// Sets your Google Cloud Platform project ID.
 	projectID := "YOUR_PROJECT_ID"
 
-	// Get a Firestore client.
-	ctx := context.Background()
 	client, err := firestore.NewClient(ctx, projectID)
 	if err != nil {
 		log.Fatalf("Failed to create client: %v", err)
 	}
+	return client
+}
 
+// [END fs_initialize]
+
+func main() {
+	// Get a Firestore client.
+	ctx := context.Background()
+	client := createClient(ctx)
 	// Close client when done.
 	defer client.Close()
```

Comment:
Looks like this line is no more in the sample. Let's add a comment (I think) into `createClient()`, like:
```
// Close client when done with
// defer client.Close()
```

Reply:
Right, thank you! I added defer to sample